tests/NexusMods.DataModel.Tests/LoadoutSynchronizerTests/IngestChangesTest.cs

```diff
@@ -58,9 +58,10 @@ public async Task RemovedFilesAreRemoved()
             })
         });
 
-        loadout.Mods.Count.Should().Be(2);
+        loadout.Mods.Count.Should().Be(3);
 
         (from mod in loadout.Mods.Values
+            where mod.Enabled == true
             from file in mod.Files.Values
             select file).Count().Should().Be(2);
```

Comment:
This should be converted into LINQ extensions: `loadout.Mods.Values.Where(...).Select(...).Count()...`

Reply:
Is there a specific advantage to using the Linq extension methods over the expression syntax?